src/routes/root/root.controller.ts

```diff
@@ -0,0 +1,9 @@
+import { Controller, Get, Res } from '@nestjs/common';
+
+@Controller()
+export class RootController {
+  @Get()
+  getIndex(@Res() res): void {
+    return res.redirect('/index.html');
+  }
+}
```

Comment:
We should test this controller. More specifically that the redirect works. Something like:

```typescript
await request(app.getHttpServer())
  .get(`/`)
  .expect(302)
  .expect((res) => {
    expect(res.get('location')).toBe('/index.html');
  });
```

Reply:
Thanks! Added in [c868631](https://github.com/safe-global/safe-client-gateway/pull/754/commits/c8686310290608c2a5a5d1d747c7cf767d231e9d)